falco.yaml

```diff
@@ -452,6 +453,75 @@ engine:
     # is the one usually passed to 'runsc --root' flag.
     root: ""
 
+##################
+# Falco captures #
+##################
+
+# [Sandbox] `capture`
+#
+# --- [Description]
+#
+# Falco captures allow you to record events and their associated data for
+# later analysis. This feature is particularly useful for debugging and
+# forensics purposes.
+#
+# Captures operate in two modes:
+#
+# 1. `rules`: Captures events only when specific rules are triggered.
+#    Enable capturing for individual rules by adding `capture: true` to the rule.
+#
+# 2. `all_rules`: Captures events when any enabled rule is triggered.
+#
+# When a capture starts, Falco records events from the moment the triggering rule
+# fires until the deadline is reached. The deadline is determined by the rule's
+# `capture_duration` if specified, otherwise the `default_duration` is used.
+# If additional rules trigger during an active capture, the deadline is extended
+# accordingly. Once the deadline expires, the capture stops and data is written
+# to a file. Subsequent captures create new files with unique names.
+#
+# Captured data is stored in files with a `.scap` extension, which can be
+# analyzed later using:
+#   falco -o engine.kind=replay -o replay.capture_file=/path/to/file.scap
+#
+# --- [Usage]
+#
+# Enable captures by setting `capture.enabled` to `true`.
+#
+# Configure `capture.path_prefix` to specify where capture files are stored.
+# Falco generates unique filenames based on timestamp and event number for
+# proper ordering. For example, with `path_prefix: /tmp/falco`, files are
+# named like `/tmp/falco_00000001234567890_00000000000000042.scap`.
+#
+# Use `capture.mode` to choose between `rules` and `all_rules` modes.
+#
+# Set `capture.default_duration` to define the default capture duration
+# in milliseconds.
+#
+# --- [Suggestion]
+#
+# When using `mode: rules`, configure individual rules to enable capture by
+# adding `capture: true` and optionally `capture_duration` to specific rules.
+# For example:
+#
+# - rule: Suspicious File Access
+#   desc: Detect suspicious file access patterns
+#   condition: >
+#     open_read and fd.name startswith "/etc/"
+#   output: >
+#     Suspicious file access (user=%user.name command=%proc.cmdline file=%fd.name)
+#   priority: WARNING
+#   capture: true
```

Comment:
Sorry, I should have mentioned this during the last review.

Maybe it makes sense to put capture below an "actions" field, in case we want to add other actions later?

Reply:
Not sure about this, let's think about it a bit. Anyway, I'd like to address this in a follow-up PR eventually, since this PR is already huge and we need to merge it soon to give it a try.